Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM node:12.20.0
+
+WORKDIR /usr/src/app
+COPY package*.json ./ 
+
+ENV COOKIE_AGE=86400000
+
+RUN yarn
+
+COPY yarn.lock ./ 
```

Comment:
This is in wrong order and a bit confusing.

1. We use yarn, so we just need `package.json` and `yarn.lock`.
2. The lock file is required to install determined version of dependencies.